sanssouci/post_hoc_bounds.py

```diff
@@ -118,8 +118,10 @@ def min_tdp(p_values, thr):
         confidence bounds on false positives using reference families.
         Annals of Statistics, 48(3), 1281-1303.
     """
-
-    return min_tp(p_values, thr) / len(p_values)
+    if len(p_values) == 0:
+        return 0
```

Comment:
Do we have a test for that case ?

Reply:
Yes, see [https://github.com/alexblnn/sanssouci.python/blob/one_sample/sanssouci/tests/test_post_hoc_bounds.py#L78-L81](url)

Reply:
Yes, see https://github.com/alexblnn/sanssouci.python/blob/one_sample/sanssouci/tests/test_post_hoc_bounds.py#L78-L81